src/components/DisplayCategoriesList.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import { Dropdown, Menu } from "semantic-ui-react";
+import axios from 'axios'
+
+const options = [
+	{ key: 1, text: "Meat", value: "meat" },
+	{ key: 2, text: "Veggie", value: "veggie" },
+	{ key: 3, text: "Glutenfree", value: "glutenfree" },
+];
+
+class DisplayCategoriesList extends Component {
+	state = {
+		options: [],
+	};
+
+	render() {
+		return (
+			<Menu compact>
+				<Dropdown
+					onChange={this.onChangeHandler}
+					id="options"
+					text="Select a category"
+					data-cy="dropdown"
+					options={options}
+					simple
+					item
+				/>
+			</Menu>
+		);
+	}
+}
+
+export default DisplayCategoriesList;
```

Comment:
format this file and you need to send in the `onChangehandler` as props, now you are calling on it but it doesn't exist in here